### PR TITLE
add terraform fmt validation

### PR DESCRIPTION
**Pull request by rphillips**
fails `make check` if terraform files are not formatted correctly.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,7 @@ release: \
 
 check:
 	@gofmt -l -s $(GOFILES) | read; if [ $$? == 0 ]; then gofmt -s -d $(GOFILES); exit 1; fi
+	@which terraform &> /dev/null && terraform fmt -check ; if [ ! $$? -eq 0 ]; then exit 1; fi
```
Comment (ericchiang) [inline Makefile]: nit: I don't think this should be conditional. it should just fail if you don't have terraform in your path

Comment (rphillips) [reply Makefile]: fixed... and rebased

**Comment by rphillips** [comment]
coreosbot run e2e

**Comment by rphillips** [comment]
coreosbot run e2e

**Comment by rphillips** [comment]
fixed jenkins and terraform pathing.

**Comment by ericchiang** [comment]
```
20:05:51 /tmp/jenkins4022084466797928612.sh: line 18: ./terraform: No such file or directory
```

Something wrong with the post build.

**Comment by rphillips** [comment]
coreosbot run e2e

**Pull request merged**